tests/lib.rs

```diff
@@ -22,14 +22,12 @@ fn test_gzip_and_zlib_encoded_and_raw_are_the_same() {
     assert_eq!(z, zstd);
 
     if let LayerData::Finite(tiles) = &c.layers[0].tiles {
-        assert_eq!(tiles.len(), 100);
-        assert_eq!(tiles[0].len(), 100);
-        assert_eq!(tiles[99].len(), 100);
-        assert_eq!(tiles[0][0].gid, 35);
-        assert_eq!(tiles[1][0].gid, 17);
-        assert_eq!(tiles[2][0].gid, 0);
-        assert_eq!(tiles[2][1].gid, 17);
-        assert!(tiles[99].iter().map(|t| t.gid).all(|g| g == 0));
+        assert_eq!(tiles.len(), 100 * 100);
+        assert_eq!(tiles[0].gid, 35);
+        assert_eq!(tiles[100].gid, 17);
+        assert_eq!(tiles[200].gid, 0);
+        assert_eq!(tiles[200 + 1].gid, 17);
+        assert!(tiles[9900..9999].iter().map(|t| t.gid).all(|g| g == 0));
```

Comment:
This is a bit hard to read, but it should be fixed soon when we add `Layer.get_tile` (#125).